spec/models/application_draft_spec.rb

```diff
@@ -327,14 +327,24 @@ def assign_project(project)
         expect(application).to have_attributes(application_draft: application_draft)
       end
 
-      it 'updates the state to :applied and sends notification email to orga' do
+      it 'updates the state to :applied and sends notification emails to orga and submitters' do
         expect { submit_application }.to change { application_draft.state }.from('draft').to('applied')
 
         submitted_application = application_draft.reload.application
 
         expect(ActionMailer::DeliveryJob).to have_been_enqueued.with(
           'ApplicationFormMailer', 'new_application', 'deliver_now', submitted_application
         )
+        expect(ActionMailer::DeliveryJob).to have_been_enqueued.with(
+          'ApplicationFormMailer', 'submitted', 'deliver_now',
+          application: submitted_application,
+          student:     application_draft.team.students.first
+        )
+        expect(ActionMailer::DeliveryJob).to have_been_enqueued.with(
```

Comment:
This looks great so far, thank you @cesswairimu!

My only concern is that issue #955 cropped up due to the fact that the students' names were assigned to the emails but the contents of the application were being assigned to the wrong student. 

Is there a way we can test the contents and make sure that the correct application information is assigned to the correct student?